packages/ui/src/components/Typography/text.tsx

```diff
@@ -18,19 +21,27 @@ export type TypographyTextProps = Omit<DefaultTextProps,
 }
 
 const Text = React.forwardRef<HTMLSpanElement, TypographyTextProps>((props, ref) => {
-    const { type, size = 'lg', ...rest } = props
+    const { type, size = 'lg', ellipsis, ...rest } = props
     const className = classNames({
         [`${TYPOGRAPHY_CLASS_PREFIX}-${type}`]: type,
         [`${TYPOGRAPHY_CLASS_PREFIX}-${size}`]: size,
     })
     // NOTE: Used wrapper destructuring to explicitly pass component props, which is marked as internal in antd
     const componentProps = { component: 'span' }
+    // NOTE: We should pass default config anyway if any prop type was provided because prefixCls should exist at result prop
+    let ellipsisProp: TypographyTextProps['ellipsis'] = false
+    if (typeof ellipsis === 'boolean' && ellipsis) {
+        ellipsisProp = DEFAULT_ELLIPSIS_CONFIG
+    } else if (typeof ellipsis === 'object') {
+        ellipsisProp = merge(DEFAULT_ELLIPSIS_CONFIG, ellipsis)
```

Comment:
Why util function for something JS can do natively?

Reply:
That's not a shallow copy of object... It's a deep copy.
Natively copy object with this case would work like this:
```js
const shallowCopy = { 
    ...{ item: { nestedValue: { nestedProp: 1 } } }, 
    ...{ item: { nestedValue: { anotherProp: 2 } } } 
}
console.log(shallowCopy)
{
    "item": {
        "nestedValue": {
            "anotherProp": 2
        }
    }
}
```

And in the case of using the utility, we will get a deep copy

```js
const deepCopy = _.merge({ item: { nestedValue: { nestedProp: 1 } } }, { item: { nestedValue: { anotherProp: 2 } } })

console.log(deepCopy)
{
    "nestedValue": {
        "nestedProp": 1,
        "anotherProp": 2
    }
}

```

Reply:
Ok, reasonable....but then DEFAULT_ELLIPSIS_CONFIG must go second, so it cannot be overwritten